.circleci/config.yml

```diff
@@ -312,22 +377,26 @@ workflows:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
       - export-package: *release-branches
-      - build-test-android:
+      - build-subtester-android: *release-branches
+      - build-subtester-ios: *release-branches
+      - build-integration-tests-android:
           <<: *release-branches
           requires:
             - export-package
-      - build-test-ios:
+      - build-integration-tests-ios:
           <<: *release-branches
           requires:
             - export-package
       - archive-ios:
           <<: *release-branches
           requires:
-            - build-test-ios
+            - build-integration-tests-ios
       - hold:
           type: approval
           requires:
-            - build-test-android
+            - build-integration-tests-android
+            - build-subtester-android
+            - build-subtester-ios
             - archive-ios
```

Comment:
`archive-ios` is currently only run for `IntegrationTests`. I think we don't need to run it as well for `Subtester`, since it seems the compilation of the APITests already happens in the `build-subtester-ios` job but lmk if you think we should do this as well.